### PR TITLE
Refactor string translation calls to use native WP translation methods - ED-681

### PR DESCRIPTION
**Pull request by udidol**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Refactor string translation calls from using elementor.translate() to WP native __() and sprintf() functions.

## Quality assurance

- [X] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: modules/history/revisions-manager.php [hunk 1]
```diff
@@ -331,42 +331,6 @@ public static function document_config( $settings, $post_id ) {
...
-	 *
-	 * Add new localized settings for the revisions manager.
-	 *
-	 * Fired by `elementor/editor/editor_settings` filter.
-	 *
-	 * @since 1.7.0
-	 * @access public
-	 * @static
-	 */
-	public static function editor_settings( $settings ) {
```
Comment (kobizz) [inline modules/history/revisions-manager.php:345]: BC

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/document/commands/base/command-disable-enable.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ export default class CommandDisableEnable extends CommandHistory {
 			changes[ id ] = settings;
 		} );
 
-		const subTitle = elementor.translate( this.constructor.getName() ) + ' ' + ElementsSettings.getSubTitle( args ),
+		const subTitle = sprintf( __( '%s', 'elementor' ), this.constructor.getName() ) + ' ' + ElementsSettings.getSubTitle( args ),
```
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/js/editor/document/commands/base/command-disable-enable.js:76]: Why not?

File: core/common/modules/connect/module.php [hunk 3]
```diff
@@ -92,6 +92,17 @@ public function init() {
...
+		add_filter( 'elementor/editor/localize_settings', [ $this, 'localize_settings' ] );
+	}
+
+	public function localize_settings( $settings ) {
+		return array_replace_recursive( $settings, [
+			'i18n' => [
+				'connect_error' => __( 'Unable to connect', 'elementor' ),
+				'connected_successfully' => __( 'Connected successfully', 'elementor' ),
+			],
+		] );
```
Comment (kobizz) [inline core/common/modules/connect/module.php]: You don't need to return back the filter and the content. Just keep an empty method and add BC doc.

File: core/dynamic-tags/manager.php [hunk 4]
```diff
@@ -39,6 +39,29 @@ public function __construct() {
...
+	 *
+	 * Fired by `elementor/editor/localize_settings` filter.
+	 *
+	 * @access public
+	 *
+	 * @param array $settings Localized settings.
+	 *
+	 * @return array Localized settings.
+	 */
+	public function localize_settings( $settings ) {
```
Comment (kobizz) [inline core/dynamic-tags/manager.php]: Same as above

File: core/upgrade/elementor-3-re-migrate-globals.php [hunk 5]
```diff
@@ -23,6 +23,15 @@ public function ajax_re_migrate_globals() {
 		return true;
 	}
 
+	public function admin_localize_settings( $settings ) {
```
Comment (kobizz) [inline core/upgrade/elementor-3-re-migrate-globals.php]: Same

File: core/upgrade/elementor-3-re-migrate-globals.php [hunk 6]
```diff
@@ -142,6 +151,7 @@ private function notice( $message ) {
 
 	public function __construct() {
 		add_action( 'elementor/admin/after_create_settings/' . Tools::PAGE_ID, [ $this, 'register_admin_tools_fields' ], 60 /* After plugins rollback */ );
+		add_action( 'elementor/admin/localize_settings', [ $this, 'admin_localize_settings' ] );
```
Comment (kobizz) [inline core/upgrade/elementor-3-re-migrate-globals.php]: Same

File: modules/history/module.php [hunk 7]
```diff
@@ -32,6 +32,44 @@ public function get_name() {
...
+	 * Fired by `elementor/editor/localize_settings` filter.
+	 *
+	 * @since 1.7.0
+	 * @access public
+	 *
+	 * @param array $settings Localized settings.
+	 *
+	 * @return array Localized settings.
+	 */
+	public function localize_settings( $settings ) {
```
Comment (kobizz) [inline modules/history/module.php]: Same

File: modules/history/module.php [hunk 8]
```diff
@@ -50,6 +88,8 @@ public function add_templates() {
 	 * @access public
 	 */
 	public function __construct() {
+		add_filter( 'elementor/editor/localize_settings', [ $this, 'localize_settings' ] );
```
Comment (kobizz) [inline modules/history/module.php]: Same

File: modules/history/revisions-manager.php [hunk 9]
```diff
@@ -331,6 +331,42 @@ public static function document_config( $settings, $post_id ) {
...
+	 *
+	 * Add new localized settings for the revisions manager.
+	 *
+	 * Fired by `elementor/editor/editor_settings` filter.
+	 *
+	 * @since 1.7.0
+	 * @access public
+	 * @static
+	 */
+	public static function editor_settings( $settings ) {
```
Comment (kobizz) [inline modules/history/revisions-manager.php]: Same

File: modules/history/revisions-manager.php [hunk 10]
```diff
@@ -353,6 +389,7 @@ public static function register_ajax_actions( Ajax $ajax ) {
 	private static function register_actions() {
 		add_action( 'wp_restore_post_revision', [ __CLASS__, 'restore_revision' ], 10, 2 );
 		add_action( 'init', [ __CLASS__, 'add_revision_support_for_all_post_types' ], 9999 );
+		add_filter( 'elementor/editor/localize_settings', [ __CLASS__, 'editor_settings' ] );
```
Comment (kobizz) [inline modules/history/revisions-manager.php]: Same

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/document/commands/base/command-disable-enable.js [hunk 11]
```diff
@@ -6,6 +6,10 @@ export default class CommandDisableEnable extends CommandHistory {
 		elementorModules.ForceMethodImplementation();
 	}
 
+	static getTitle() {
```
Comment (kobizz) [inline assets/dev/js/editor/document/commands/base/command-disable-enable.js:9]: Actually, we don't need this method. We can rely on `getName` when the return value is changed.

File: assets/dev/js/editor/document/dynamic/commands/base/disable-enable.js [hunk 12]
```diff
@@ -5,6 +5,10 @@ export default class DisableEnable extends CommandDisableEnable {
 		return 'dynamic';
 	}
 
+	static getTitle() {
```
Comment (kobizz) [inline assets/dev/js/editor/document/dynamic/commands/base/disable-enable.js:8]: Same as above

File: assets/dev/js/editor/document/commands/base/command-disable-enable.js [hunk 13]
```diff
@@ -73,7 +77,7 @@ export default class CommandDisableEnable extends CommandHistory {
 			changes[ id ] = settings;
 		} );
 
-		const subTitle = elementor.translate( this.constructor.getName() ) + ' ' + ElementsSettings.getSubTitle( args ),
+		const subTitle = this.constructor.getTitle() + ' ' + ElementsSettings.getSubTitle( args ),
```
Comment (kobizz) [inline assets/dev/js/editor/document/commands/base/command-disable-enable.js:80]: We need to change the returned value from `getName` to be translated there.

File: includes/controls/deprecated-notice.php [hunk 14]
```diff
@@ -45,12 +45,12 @@ public function content_template() {
 		<span class="elementor-control-title">{{{ data.label }}}</span>
 		<#
 		}
-		let notice = elementor.translate( 'deprecated_notice', [ data.widget, data.plugin, data.since ] );
+		let notice = sprintf( wp.i18n.__( 'The <strong>%1$s</strong> widget has been deprecated since %2$s %3$s.', 'elementor' ), data.widget, data.plugin, data.since );
```
Comment (kobizz) [inline includes/controls/deprecated-notice.php:48]: 1. Use `wp.i18n.sprintf`.
2. You can translate the string itself from the php.

File: includes/controls/deprecated-notice.php [hunk 15]
```diff
@@ -45,12 +45,12 @@ public function content_template() {
...
 		if ( data.replacement ) {
-			notice += '<br>' + elementor.translate( 'deprecated_notice_replacement', [ data.replacement ] );
+			notice += '<br>' + sprintf( wp.i18n.__( 'It has been replaced by <strong>%1$s</strong>.', 'elementor' ), data.replacement );
 		}
 		if ( data.last ) {
-			notice += '<br>' + elementor.translate( 'deprecated_notice_last', [ data.widget, data.plugin, data.last ] );
+			notice += '<br>' + sprintf( wp.i18n.__( 'Note that %1$s will be completely removed once %2$s %3$s is released.', 'elementor' ), data.widget, data.plugin, data.last );
```
Comment (kobizz) [inline includes/controls/deprecated-notice.php:53]: Same here

File: modules/history/views/revisions-panel-template.php [hunk 16]
```diff
@@ -28,12 +28,17 @@
...
+		revisions_disabled_1 = wp.i18n.__( 'It looks like the post revision feature is unavailable in your website.', 'elementor' ),
+		revisions_disabled_2 = sprintf( wp.i18n.__( 'Learn more about <a target="_blank" href="%s">WordPress revisions</a>', 'elementor' ), 'https://go.elementor.com/wordpress-revisions/' ); /* translators: %s: Codex URL */
+	#>
 	<img class="elementor-nerd-box-icon" src="<?php echo ELEMENTOR_ASSETS_URL . 'images/information.svg'; ?>" />
 	<div class="elementor-nerd-box-title"><?php echo __( 'No Revisions Saved Yet', 'elementor' ); ?></div>
-	<div class="elementor-nerd-box-message">{{{ elementor.translate( elementor.config.document.revisions.enabled ? 'no_revisions_1' : 'revisions_disabled_1' ) }}}</div>
-	<div class="elementor-nerd-box-message">{{{ elementor.translate( elementor.config.document.revisions.enabled ? 'no_revisions_2' : 'revisions_disabled_2' ) }}}</div>
+	<div class="elementor-nerd-box-message">{{{ elementor.config.document.revisions.enabled ? no_revisions_1 : revisions_disabled_1 }}}</div>
+	<div class="elementor-nerd-box-message">{{{ elementor.config.document.revisions.enabled ? no_revisions_2 : revisions_disabled_2 }}}</div>
 </script>
```
Comment (kobizz) [inline modules/history/views/revisions-panel-template.php:42]: You can translate all values in PHP side

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/kits/assets/js/repeater.js [hunk 17]
```diff
@@ -11,15 +11,15 @@ export default class extends Repeater {
 	templateHelpers() {
 		const templateHelpers = super.templateHelpers();
 
-		templateHelpers.addButtonText = elementor.translate( 'custom_colors' === this.model.get( 'name' ) ? 'add_color' : 'add_style' );
+		templateHelpers.addButtonText = sprintf( __( 'Add %s', 'elementor' ), 'custom_colors' === this.model.get( 'name' ) ? __( 'Color', 'elementor' ) : __( 'Style', 'elementor' ) );
```
Comment (kobizz) [inline core/kits/assets/js/repeater.js:14]:
```suggestion
		templateHelpers.addButtonText = 'custom_colors' === this.model.get( 'name' ) ? __( 'Add Color', 'elementor' ) : __( 'Add Style', 'elementor' );
```

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — COMMENTED** [review]
Find out why is the JS-Qunit failing.

**Review by kobizz — DISMISSED** [review]

**Review by kobizz — DISMISSED** [review]

File: core/common/modules/connect/module.php [hunk 18]
```diff
@@ -92,17 +93,15 @@ public function init() {
...
-			'i18n' => [
-				'connect_error' => __( 'Unable to connect', 'elementor' ),
-				'connected_successfully' => __( 'Connected successfully', 'elementor' ),
-			],
-		] );
+	/**
+	 * @deprecated 3.1.0
+	 */
+	public function localize_settings() {
+		Plugin::$instance->common->get_component( 'devTools' )->deprecation->deprecated_function( __METHOD__, '3.1.0' );
```
Comment (kobizz) [inline core/common/modules/connect/module.php:102]: Change all occurrences to the new format

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/common/modules/connect/module.php (hunk 18 above)
Comment (udidol) [reply core/common/modules/connect/module.php]: Done

**Review by kobizz — DISMISSED** [review]

File: core/admin/feedback.php [hunk 19]
```diff
@@ -70,17 +68,6 @@ public function enqueue_feedback_dialog_scripts() {
 		wp_enqueue_script( 'elementor-admin-feedback' );
 	}
 
-	/**
-	 * @since 2.3.0
-	 * @access public
-	 */
-	public function localize_feedback_dialog_settings( $localized_settings ) {
```
Comment (kobizz) [inline core/admin/feedback.php:77]: Return back this method as BC

File: core/common/app.php [hunk 20]
```diff
@@ -5,6 +5,7 @@
 use Elementor\Core\Common\Modules\Ajax\Module as Ajax;
 use Elementor\Core\Common\Modules\Finder\Module as Finder;
 use Elementor\Core\Common\Modules\Connect\Module as Connect;
+use Elementor\Core\Common\Modules\DevTools\Module as Devtools;
```
Comment (kobizz) [inline core/common/app.php:8]: This is not supposed to be here

File: core/common/app.php [hunk 21]
```diff
@@ -61,6 +62,7 @@ public function init_components() {
 		}
 
 		$this->add_component( 'connect', new Connect() );
+		$this->add_component( 'devTools', new Devtools() );
```
Comment (kobizz) [inline core/common/app.php:65]: Same as above

File: core/common/assets/js/common.js [hunk 22]
```diff
@@ -4,6 +4,7 @@ import Debug from './utils/debug';
 import Ajax from 'elementor-common-modules/ajax/assets/js/ajax';
 import Finder from 'elementor-common-modules/finder/assets/js/finder';
 import Connect from 'elementor-common-modules/connect/assets/js/connect';
+import DevTools from 'elementor-common-modules/dev-tools/assets/js/dev-tools';
```
Comment (kobizz) [inline core/common/assets/js/common.js:7]: Same for all changes in this file

File: core/common/modules/dev-tools/assets/js/dev-tools.js [hunk 23]
```diff
@@ -0,0 +1,17 @@
+export default class extends elementorModules.Module {
```
Comment (kobizz) [inline core/common/modules/dev-tools/assets/js/dev-tools.js:1]: Same about this file

File: core/common/modules/dev-tools/deprecation.php [hunk 24]
```diff
@@ -0,0 +1,188 @@
+<?php
```
Comment (kobizz) [inline core/common/modules/dev-tools/deprecation.php:1]: Same here

File: core/common/modules/dev-tools/module.php [hunk 25]
```diff
@@ -0,0 +1,31 @@
+<?php
```
Comment (kobizz) [inline core/common/modules/dev-tools/module.php:1]: Same

File: includes/plugin.php [hunk 26]
```diff
@@ -21,6 +21,7 @@
 use Elementor\Core\Logger\Manager as Log_Manager;
 use Elementor\Modules\System_Info\Module as System_Info_Module;
 use Elementor\Data\Manager as Data_Manager;
+use Elementor\Core\Common\Modules\DevTools\Module as Dev_Tools;
```
Comment (kobizz) [inline includes/plugin.php:24]: Same for this file

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/plugin.php (hunk 26 above)
Comment (udidol) [reply includes/plugin.php]: Done.

File: core/common/modules/dev-tools/deprecation.php (hunk 24 above)
Comment (udidol) [reply core/common/modules/dev-tools/deprecation.php]: Done.

File: core/common/modules/dev-tools/assets/js/dev-tools.js (hunk 23 above)
Comment (udidol) [reply core/common/modules/dev-tools/assets/js/dev-tools.js]: Done.

File: core/common/assets/js/common.js (hunk 22 above)
Comment (udidol) [reply core/common/assets/js/common.js]: Done.

File: core/common/app.php (hunk 21 above)
Comment (udidol) [reply core/common/app.php]: Done.

File: core/common/app.php (hunk 20 above)
Comment (udidol) [reply core/common/app.php]: Done.

File: core/admin/feedback.php (hunk 19 above)
Comment (udidol) [reply core/admin/feedback.php:77]: Done.

**Review by kobizz — DISMISSED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js [hunk 27]
```diff
@@ -132,7 +132,7 @@ export const Create = () => {
 					eInnerSection.view.children.forEach( ( el ) => innerSectionColumnsIds.push( el.model.id ) );
 
 					// Exist in history.
-					HistoryHelper.inHistoryValidate( assert, historyItem, 'add', 'inner_section' );
+					HistoryHelper.inHistoryValidate( assert, historyItem, 'add', 'Inner Section' );
```
Comment (matipojo) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js:135]: It shouldn't be changed, here and next.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/document/commands/base/command-disable-enable.js [hunk 28]
```diff
@@ -2,10 +2,6 @@ import CommandHistory from './command-history';
 import ElementsSettings from 'elementor-document/elements/commands/settings';
 
 export default class CommandDisableEnable extends CommandHistory {
-	static getName() {
```
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/js/editor/document/commands/base/command-disable-enable.js:5]: @kobizz ?

File: assets/dev/js/editor/elements/views/widget.js [hunk 29]
```diff
@@ -56,18 +56,18 @@ WidgetView = BaseElementView.extend( {
 			editTools = {};
 
 		editTools.edit = {
-			title: elementor.translate( 'edit_element', [ elementData.title ] ),
+			title: sprintf( __( 'Edit %s', 'elementor' ), elementData.title ),
```
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/js/editor/elements/views/widget.js:60]: Done.

File: core/kits/assets/js/repeater-row.js [hunk 30]
```diff
@@ -58,17 +58,18 @@ export default class extends RepeaterRow {
 		}
 
 		if ( isColor || isPopoverToggle ) {
-			const removeButtons = this.getDisabledRemoveButtons();
+			const removeButtons = this.getDisabledRemoveButtons(),
+				capitalizedGlobalType = elementorCommon.helpers.upperCaseWords( globalType );
```
[the remark this answers is not in the document]
Comment (udidol) [reply core/kits/assets/js/repeater-row.js:62]: Please explain @matipojo

File: core/kits/assets/js/repeater-row.js [hunk 31]
```diff
@@ -81,15 +82,17 @@ export default class extends RepeaterRow {
 	}
 
 	onRemoveButtonClick() {
-		const globalType = this.ui.removeButton.data( 'e-global-type' );
+		const globalType = this.ui.removeButton.data( 'e-global-type' ),
+			capitalizedGlobalType = elementorCommon.helpers.upperCaseWords( globalType ),
+			translatedMessage = sprintf( __( 'You\'re about to delete a Global %s. Note that if it\'s being used anywhere on your site, it will inherit a default typography.', 'elementor' ), capitalizedGlobalType, 'Font' === capitalizedGlobalType ? 'typography' : 'color' );
```
[the remark this answers is not in the document]
Comment (udidol) [reply core/kits/assets/js/repeater-row.js]: Fixed. Was missing another %s in the last word of the string

File: includes/controls/deprecated-notice.php (hunk 14 above)
[the remark this answers is not in the document]
Comment (udidol) [reply includes/controls/deprecated-notice.php:48]: its a `content_template()`, `__` is not globally available in the frontend but `wp.18n.__` is

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js (hunk 27 above)
Comment (udidol) [reply tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js:135]: What do you mean next? @matipojo

File: modules/history/assets/js/component.js [hunk 32]
```diff
@@ -7,8 +7,8 @@ export default class Component extends ComponentBase {
 
 	defaultTabs() {
 		return {
-			actions: { title: elementor.translate( 'actions' ) },
-			revisions: { title: elementor.translate( 'revisions' ) },
+			actions: { title: __( 'actions', 'elementor' ) },
+			revisions: { title: __( 'revisions', 'elementor' ) },
```
[the remark this answers is not in the document]
Comment (udidol) [reply modules/history/assets/js/component.js:11]: Fixed.

File: includes/controls/deprecated-notice.php (hunk 14 above)
[the remark this answers is not in the document]
Comment (kobizz) [reply includes/controls/deprecated-notice.php:48]: Use `wp.i18n.sprintf`

[the remark this answers is not in the document]
Comment (udidol) [reply includes/controls/deprecated-notice.php]: Done @kobizz

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 33]
```diff
@@ -86,7 +86,7 @@ module.exports = elementorModules.ViewModule.extend( {
 				attributes: {
 					tabindex: 0,
 					role: 'button',
-					'aria-label': elementorFrontend.config.i18n.close + ' (Esc)',
+					'aria-label': __( 'Close', 'elementor' ) + ' (Esc)',
```
[the remark this answers is not in the document]
Comment (kobizz) [reply assets/dev/js/frontend/utils/lightbox/lightbox.js:89]: @udidol Return it back to the `i18n` config

File: core/kits/assets/js/repeater-row.js (hunk 30 above)
[the remark this answers is not in the document]
Comment (kobizz) [reply core/kits/assets/js/repeater-row.js:62]: @udidol Add translations. For example: `globalType = __( 'Font', 'elementor' );`

File: assets/dev/js/editor/regions/panel/pages/elements/views/element.js [hunk 34]
```diff
@@ -50,9 +50,12 @@ module.exports = Marionette.ItemView.extend( {
...
 
 	onMouseDown: function() {
+		const title = this.model.get( 'title' );
+
 		elementor.promotion.showDialog( {
-			headerMessage: elementor.translate( 'element_promotion_dialog_header', [ this.model.get( 'title' ) ] ),
-			message: elementor.translate( 'element_promotion_dialog_message', [ this.model.get( 'title' ) ] ),
+			/* translators: %s: Widget Title. */
+			headerMessage: sprintf( __( '%s Widget', 'elementor' ), title ),
+			message: sprintf( __( 'Use %s widget and dozens more pro features to extend your toolbox and build sites faster and better.', 'elementor' ), title ),
```
Comment (kobizz) [inline assets/dev/js/editor/regions/panel/pages/elements/views/element.js:58]: Add translators

**Review by kobizz — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js [hunk 35]
```diff
@@ -252,7 +252,7 @@ export const Create = () => {
 					} );
 
 					// Exist in history.
-					HistoryHelper.inHistoryValidate( assert, historyItem, 'add', 'inner_section' );
+					HistoryHelper.inHistoryValidate( assert, historyItem, 'add', 'Inner Section' );
```
Comment (kobizz) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js:255]: Do it also at line 135

**Review by kobizz — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js (hunk 35 above)
Comment (udidol) [reply tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js:255]: Done.

File: assets/dev/js/editor/regions/panel/pages/elements/views/element.js (hunk 34 above)
Comment (udidol) [reply assets/dev/js/editor/regions/panel/pages/elements/views/element.js:59]: Done.

File: core/kits/assets/js/repeater-row.js [hunk 36]
```diff
@@ -81,15 +82,17 @@ export default class extends RepeaterRow {
 	}
 
 	onRemoveButtonClick() {
-		const globalType = this.ui.removeButton.data( 'e-global-type' );
+		const globalType = this.ui.removeButton.data( 'e-global-type' ),
+			capitalizedGlobalType = elementorCommon.helpers.upperCaseWords( globalType ),
```
[the remark this answers is not in the document]
Comment (udidol) [reply core/kits/assets/js/repeater-row.js]: Done.

File: core/kits/assets/js/repeater-row.js (hunk 30 above)
[the remark this answers is not in the document]
Comment (udidol) [reply core/kits/assets/js/repeater-row.js]: Done.

File: assets/dev/js/frontend/utils/lightbox/lightbox.js (hunk 33 above)
[the remark this answers is not in the document]
Comment (udidol) [reply assets/dev/js/frontend/utils/lightbox/lightbox.js]: Done.

File: includes/frontend.php [hunk 37]
```diff
@@ -1153,17 +1155,6 @@ protected function get_init_settings() {
...
-				'shareOnTwitter' => __( 'Share on Twitter', 'elementor' ),
-				'pinIt' => __( 'Pin it', 'elementor' ),
-				'download' => __( 'Download', 'elementor' ),
-				'downloadImage' => __( 'Download image', 'elementor' ),
-				'fullscreen' => __( 'Fullscreen', 'elementor' ),
-				'zoom' => __( 'Zoom', 'elementor' ),
-				'share' => __( 'Share', 'elementor' ),
-				'playVideo' => __( 'Play Video', 'elementor' ),
-				'previous' => __( 'Previous', 'elementor' ),
-				'next' => __( 'Next', 'elementor' ),
```
Comment (kobizz) [inline includes/frontend.php:1166]: Change all instances to use `wp.i18n`

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/frontend.php (hunk 37 above)
Comment (udidol) [reply includes/frontend.php:1166]: Done.

**Review by kobizz — DISMISSED** [review]

File: assets/dev/js/frontend/utils/lightbox/lightbox.js [hunk 38]
```diff
@@ -86,7 +86,7 @@ module.exports = elementorModules.ViewModule.extend( {
 				attributes: {
 					tabindex: 0,
 					role: 'button',
-					'aria-label': elementorFrontend.config.i18n.close + ' (Esc)',
+					'aria-label': wp.i18n.__( 'Close', 'elementor' ) + ' (Esc)',
```
Comment (matipojo) [inline assets/dev/js/frontend/utils/lightbox/lightbox.js:89]: Export `wp.i18n.__` to a global like on the editor.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/assets/js/repeater-row.js [hunk 39]
```diff
@@ -81,15 +83,18 @@ export default class extends RepeaterRow {
 	}
 
 	onRemoveButtonClick() {
-		const globalType = this.ui.removeButton.data( 'e-global-type' );
+		const globalType = this.ui.removeButton.data( 'e-global-type' ),
+			/* translators: First %s: Font/Color. Second %s: typography/color */
+			translatedMessage = sprintf( __( 'You\'re about to delete a Global %s. Note that if it\'s being used anywhere on your site, it will inherit a default %s.', 'elementor' ), globalType, __( 'Font', 'elementor' ) === globalType ? 'typography' : 'color' );
```
[the remark this answers is not in the document]
Comment (udidol) [reply core/kits/assets/js/repeater-row.js]: Fixed.

**Review by kobizz — DISMISSED** [review]

**Review by matipojo — DISMISSED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**